src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  FC,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+
+import { useParams } from 'react-router-dom';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { PeopleTable } from '../components/PeopleTable';
+import { getPeople } from '../api';
+
+export const PeoplePage: FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  const { personSlug = '' } = useParams();
```

Comment:
The type of 'personSlug' in useParams should be explicitly specified. Consider using 'useParams<{ personSlug?: string }>()'